### PR TITLE
Update to holochain v25 + updated anchor code from Clutter

### PR DESCRIPTION
**Pull request by ddd-mtl**
#45
https://waffle.io/holochain/org/cards/5b0c036ca2f2bc001b6b91f5

### CODE REVIEW

File: dna/cointoss/cointoss.js
Comment (zippy) [inline dna/cointoss/cointoss.js:1]: This line comes from the code that got moved into core around allowing the app designer to choose throwing errors or returning them.  We should remove this otherwise it will confuse folks.

File: dna/cointoss/cointoss.js
Comment (zippy) [inline dna/cointoss/cointoss.js:37]: I think we should remove this mermaid debugging.  I don't think it adds anything to cointoss.

**Review by zippy — CHANGES_REQUESTED** [review]
Great work, besides those two requests, I think it's ready to merge.

**Comment by philipbeadle** [comment]
I merged it as I added travis to develop and it works fine

**Comment by lucksus** [comment]
This PR was merged without approval. I only see @zippy requesting changes. We say in the Playbooklet that "Pull requests need approval of at least two other team members before they are allowed to be merged."

https://docs.google.com/document/d/1Pzs9aXk1eQfyMWbwDyHgCz6kiHs_FpHjscaZyUlpei8/edit

**Comment by ddd-mtl** [comment]
I made the changes zippy requested on develop. Should I do a new pull request? It seems I can't add to this PR anymore now that it is merged.

**Comment by zippy** [comment]
Just go ahead an merge it!

**Pull request merged**
